tests/_support/Step/Acceptance/Administrator/LoginSteps.php

```diff
@@ -0,0 +1,36 @@
+<?php
+namespace Step\Acceptance\Administrator;
+
+class LoginSteps extends \AcceptanceTester
+{
+
+	private $usernameFieldId = ['id' => 'mod-login-username'];
+
+	private $passwordFieldId = ['id' => 'mod-login-password'];
+
+	private $pageTitleClass = ['class' => 'page-title'];
+
+	private $loginButtonPath = ['xpath' => "//button[contains(normalize-space(), 'Log in')]"];
```

Comment:
That xpath is probably really slow :( As we don't have a unique identifier there, we probably should try using the name ($I->click('Log in');) What do you think?

Or you can use $I->click('class' => 'btn-primary'); here.. Even if it is generic, it's the only one on that page.